### PR TITLE
go rewrite healthcare

### PR DESCRIPTION
**Pull request by c2thorn**
healthcare needs to be omitted from automatic refresh because of `exact_version` YAML attribute

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/products/healthcare/go_ConsentStore.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  method_name_separator: ':'
+  parent_resource_attribute: 'consent_store_id'
+  base_url: '{{%dataset}}/consentStores/{{name}}'
+  example_config_body: 'templates/terraform/iam/go/iam_attributes.go.tmpl'
+  self_link: '{{%dataset}}/consentStores/{{name}}'
+  import_format:
+    - '{{%dataset}}/consentStores/{{name}}'
+    - '{{name}}'
+custom_code:
+skip_sweeper: true
```
Comment (NickElliot) [inline mmv1/products/healthcare/go_ConsentStore.yaml:46]:
```suggestion
exclude_sweeper: true
```

File: mmv1/products/healthcare/go_DicomStore.yaml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+import_format:
+  - '{{dataset}}/dicomStores/{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  decoder: 'templates/terraform/decoders/go/long_name_to_self_link.go.tmpl'
+  custom_import: 'templates/terraform/custom_import/go/healthcare_dicom_store.go.tmpl'
+skip_sweeper: true
```
Comment (NickElliot) [inline mmv1/products/healthcare/go_DicomStore.yaml:40]:
```suggestion
exclude_sweeper: true
```

File: mmv1/products/healthcare/go_FhirStore.yaml [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+import_format:
+  - '{{dataset}}/fhirStores/{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  decoder: 'templates/terraform/decoders/go/long_name_to_self_link.go.tmpl'
+  custom_import: 'templates/terraform/custom_import/go/healthcare_fhir_store.go.tmpl'
+skip_sweeper: true
```
Comment (NickElliot) [inline mmv1/products/healthcare/go_FhirStore.yaml:40]:
```suggestion
exclude_sweeper: true
```

File: mmv1/products/healthcare/go_FhirStore.yaml [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+      bq_dataset_name: 'bq_example_dataset'
+    test_vars_overrides:
+      'policyChanged': ' acctest.BootstrapPSARoles(t, "service-", "gcp-sa-healthcare", []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})'
+  - name: 'healthcare_fhir_store_notification_config'
+    primary_resource_id: 'default'
+    vars:
+      dataset_name: 'example-dataset'
+      fhir_store_name: 'example-fhir-store'
+      pubsub_topic: 'fhir-notifications'
+    skip_docs: true
```
Comment (NickElliot) [inline mmv1/products/healthcare/go_FhirStore.yaml:63]: oh I should edit these too, I'll pre emptively edit these then do another passover
```suggestion
    exclude_docs: true
```

File: mmv1/products/healthcare/go_Hl7V2Store.yaml [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+import_format:
+  - '{{dataset}}/hl7V2Stores/{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  decoder: 'templates/terraform/decoders/go/long_name_to_self_link.go.tmpl'
+  custom_import: 'templates/terraform/custom_import/go/healthcare_hl7_v2_store.go.tmpl'
+skip_sweeper: true
```
Comment (NickElliot) [inline mmv1/products/healthcare/go_Hl7V2Store.yaml:40]:
```suggestion
exclude_sweeper: true
```

**Review by NickElliot — APPROVED** [review]
let me know what you think about the skip_docs rename

**Comment by c2thorn** [comment]
Oh, I should have rebased so I get the rename changes

**Comment by c2thorn** [comment]
Thanks for making the suggestions, that should be all that's needed to fix

**Pull request merged**